src/lambda/src/clients/aws-client.ts

```diff
@@ -0,0 +1,40 @@
+import { CodeBuild } from '@aws-sdk/client-codebuild';
+import { CodePipeline } from '@aws-sdk/client-codepipeline';
+
+export class AWSClient {
```

Comment:
It is for both codebuild and codepipeline client calls, but since at least in AWS console they are coupled under CodePipelines service, AWSCodePipelineClient name can be sensible.